### PR TITLE
[urtsi] Fixed leading spaces in translations

### PR DESCRIPTION
**Pull request by cweitkamp**
- Fixed leading spaces in translations

Signed-off-by: Christoph Weitkamp <github@christophweitkamp.de>

### CODE REVIEW

File: bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi.properties [hunk 1]
```diff
@@ -1,18 +1,22 @@
 # binding
+
+binding.urtsi.name = Somfy URTSI II Binding
```
Comment (lolodomo) [inline bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi.properties:3]: I suggest a change in binding.xml to use the same approach for label and description, either both using `@text` or none.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi.properties (hunk 1 above)
Comment (cweitkamp) [reply bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi.properties]: Done.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
@cweitkamp : thank you, I let you fix the German translation (leading spaces).

**Pull request merged**